docs/dev/table/streaming/query_configuration.md

```diff
@@ -135,6 +160,16 @@ val qConfig: StreamQueryConfig = ???
 // set idle state retention time: min = 12 hours, max = 24 hours
 qConfig.withIdleStateRetentionTime(Time.hours(12), Time.hours(24))
 
+{% endhighlight %}
+</div>
+<div data-lang="python" markdown="1">
+{% highlight python %}
+
+q_config = ...
```

Comment:
Add a comment description that the type of variable q_config is "StreamQueryConfig"